the-collab-lab/tcl-71-smart-shopping-list
# Alerts in-app

## Description

This PR displays alert messages from adding items or sharing a list in-app, instead of with the browser's native alert() function.

## Related Issue

closes #73

## Acceptance Criteria

- [x] When sharing the list with another user, all messages are displayed below the component, instead of in an alert message.
- [x] When adding an item, the confirmation message is desplayed below the component, instead of in an alert message.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: New feature     |
|  ✓    | :art:  Enhancement |

## Updates

### Before

![image](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/86715948/cc2a11c9-c9f2-43b5-a767-57b2b2bae82c)

![image](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/86715948/3f06c55e-e928-4020-b072-3a5b52d99182)

### After

![image](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/86715948/d9e15830-af21-490e-8996-d2ceaee03a89)

![image](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/86715948/7a314af8-0315-42dc-aa6b-7c8501dd8520)


## Testing Steps / QA Criteria

- `git pull`
- `git checkout alert-messages`
- Add items and share a list, observe resulting message.
